randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,36 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          Morph Ball
+          Any of the following:
+              All of the following:
+                  Use Spin Boost
+                  Movement (Intermediate) or Slide Jump (Beginner)
```

Comment:
Have you even tried this with a slide jump? it doesnt make it easier at all cause of the crumble blocks. Your slide just gets eaten up by them

Reply:
The pitfall blocks slow down your slide, making the timing very easy.